components/clm/src/main/histFileMod.F90

```diff
@@ -1856,7 +1864,8 @@ subroutine htape_create (t, histrest)
 
     ! Global compressed dimensions (not including non-land points)
     call ncd_defdim(lnfid, trim(nameg), numg, dimid)
-    call ncd_defdim(lnfid, trim(namet), numt, dimid)
+    !call ncd_defdim(lnfid, trim(namet), max_topounits, dimid) !TKT topounit dimension needs to be max_topounits
```

Comment:
Remove this commented out line

Reply:
Thanks! Commented line has been removed.